src/frontend/query_protocol.ml

```diff
@@ -103,6 +103,13 @@ type syntax_doc_result =
     documentation : string
 }
 
+type ppx_expand_result = 
+{
+  code : string;
```

Comment:
Sorry, that really neatpicky, but I would prefer another name like `source` or `ppxed_source`, code is a bit vague.